.github/workflows/release.yml

```diff
@@ -30,3 +30,38 @@ jobs:
             target/*.tar.gz
             target/rpm/jcliff/RPMS/noarch/*.rpm
           body: "Release ${{ steps.get_version.outputs.TAG_VERSION }}"
+      - name: Homebrew Release
+        env:
+          HOMEBREW_TAP_REPO: ${{ secrets.HOMEBREW_REPOSITORY }}
+          BREW_USER: ${{ secrets.BREW_USER }}
+          BREW_TOKEN: ${{ secrets.BREW_TOKEN }}
+        run: |
+          set -e
+
+          JCLIFF_TAG=${{ steps.get_version.outputs.TAG_VERSION }}
+          HOMEBREW_TAP_REPO="${HOMEBREW_TAP_REPO:-redhat-cop/homebrew-redhat-cop}"
+          REPO_NAME=$(basename $HOMEBREW_TAP_REPO)
+
+          SHA256=`sha256sum target/jcliff-${JCLIFF_TAG:1}-dist.tar.gz | cut -f1 -d ' '`
+          URL="https://github.com/bserdar/jcliff/releases/download/${JCLIFF_TAG}/jcliff-${JCLIFF_TAG:1}-dist.tar.gz"
+
+          rm -rf $REPO_NAME
```

Comment:
=> "${REPO_NAME}"

especially here. Imagine:

REPO_NAME='a_file /' => boom root is erased!